devicetypes/timevalve-gaslock-t-08/timevalve-smart.src/timevalve-smart.groovy

```diff
@@ -0,0 +1,281 @@
+metadata {
+        definition (name: "Timevalve Smart", namespace: "timevalve.gaslock.t-08", author: "ruinnel") {
+                capability "Valve"
+                capability "Refresh"
+                capability "Battery"
+                capability "Notification"
+				capability "Configuration"
+                capability "Temperature Measurement"
+                
```

Comment:
Consider adding logic for auto temperature localization based on hub settings (Celsius vs Fahrenheit) and proper conversion for correct temperature display